ai_chatbots/tools.py

```diff
@@ -268,7 +273,12 @@ def get_video_transcript_chunk(q: str, state: Annotated[dict, InjectedState]) ->
 
     log.debug("Searching MIT API with params: %s", params)
     try:
-        response = requests.get(url, params=params, timeout=30)
+        response = requests.get(
+            url,
+            params=params,
+            headers={"Authorization": f"Bearer {settings.LEARN_ACCESS_TOKEN}"},
+            timeout=30,
+        )
```

Comment:
Minor suggestion, take or leave: this is repeated a few times, maybe it could be placed in a common function?

```python
def request_with_token(url, params, timeout: int = 30):
    return requests.get(
            url,
            params=params,
            headers={"Authorization": f"Bearer {settings.LEARN_ACCESS_TOKEN}"},
            timeout=timeout,
        )
```